src/components/Sign/SignForm.tsx

```diff
@@ -33,21 +37,55 @@ export default function SignForm({ type }: SignFormProp) {
     register,
     handleSubmit,
     getValues,
+    setError,
     formState: { errors },
   } = useForm<SignForm>({ mode: 'onBlur' });
+  const router = useRouter();
 
-  const handlePasswordToggle = () => {
+  const handlePasswordToggle = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
     setIsPasswordShown(!isPasswordShown);
   };
-  const handlePasswordConfirmToggle = () => {
+  const handlePasswordConfirmToggle = (
+    e: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    e.preventDefault();
     setIsPasswordConfirmShown(!isPasswordConfirmShown);
   };
 
+  const handleSignIn = async (data: SignForm) => {
+    delete data.passwordConfirm;
```

Comment:
이 로직으로 구현하면 불변성이 안 지켜지는 것으로 보여요!
다음처럼 rest 문법을 사용하는 건 어떨까요?
```suggestion
    const {passwordConfirm, ...payload} = data;
```